gpu/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "lambdaworks-gpu"
```

Comment:
Is it going to be shared between multiple GPUs ? Else I'd call this lambdaworks-metal. Or lambdaworks-metal-gpu